### PR TITLE
support sysdate time function 

### PR DESCRIPTION
**Pull request by siddontang**

### CODE REVIEW

File: parser/scanner.l [hunk 1]
```diff
@@ -689,6 +690,8 @@ sys_var		"@@"(({global}".")|({session}".")|{local}".")?{ident}
 			return substring
 {sum}			lval.item = string(l.val)
 			return sum
+{sysdate}		lval.item = string(l.val)
+			return sysDate
```
Comment (qiuyesuifeng) [inline parser/scanner.l]: sysdate or sysDate?

Comment (siddontang) [inline parser/scanner.l]: we define sysDate in parser, so here it is sysDate

**Comment by siddontang** [comment]
PTAL @qiuyesuifeng

**Comment by coocood** [comment]
LGTM

**Pull request merged**